lib/cloudregister/registerutils.py

```diff
@@ -904,6 +961,22 @@ def is_registered(smt):
     return None
 
 
+# ----------------------------------------------------------------------------
+def is_scc_connected():
+    """If any repo url points to suse.com then at least some of the
+       repositories come from SCC and the SCCcredentials file is
+       expected to have the system credentials for SCC access."""
```

Comment:
If the SCC crendentials file is what we're after, shouldn't the check verify it is present as well? I can easily imagine a misconfigured system where there is a repo with baseurl of suse.com, but /etc/zypp/credentials.d/SCCcredentials has been removed.

Reply:
Well that is handled here 

https://github.com/SUSE-Enceladus/cloud-regionsrv-client/pull/52/files#diff-84460adc47c95f16ae220daac1882a7bc19b595d1ba01650e286b666ef14f2a4R1113

I don't think an explicit check is necessary as the result would be the same. If the file does not exist we get an auth object that is basically `NULL` and then SCC when we call `DELETE` will return something other than `204` and as such we will continue to proceed to remove any references to `suse.com` and the system if tracked by some means in SCC would remain tracked and would require manual clean up. We probably need a better error message.